examples/hello-world/README.md

```diff
@@ -3,26 +3,13 @@ React Intl Hello World Example
 
 This is a very simple — yet runnable app — showing how to use React Intl to format message (string) that contains a placeholder, plural, and number.
 
-## Running
+**This project template was built with [Create React App](https://github.com/facebookincubator/create-react-app).**
```

Comment:
This looks too much like advertising here 😄 
I think we should be more humble and just link to the [user guide](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md) instead.